### DISCUSSION TITLE
Query Editor/Table Background Color Theme

### DISCUSSION THREAD

**Discussion by krashstudios** [microsoft/azuredatastudio#23231] [Q&A]
Currently the results table blends into the editor and share the same hook 'editor.background'
Is there a seperate hook to change the results table cell background color?
I have found queryEditor.nullBackground to change the NULL value background color but would like to change the entire table background as well as the cell selection color.
Something like 'table.bodyBackground' as there is table.headerBackground

**Comment by erinstellato-ms** [comment]
Hey @krashstudios - thank you for starting this thread!  Would you mind sharing a screenshot or two so we are clear on the blending that you mention for the results table and the editor.  If you can call out where the differentiation between panes is not clear for you, and what specific area you would like to independently change, that would be of great help.  I recognize this may seem very obvious to you!  But I am very much a "visual person" and I do not want to assume that I know exactly what you're talking about.  I find that having examples helps me confirm that my understanding matches what you are asking for.  Thank you!

**Reply by krashstudios** [reply]
@erinstellato-ms Absolutely!
I'd like to be able to change the results cell background color as well as the selected cell color. Not sure if there aren't any hooks or if I just can't find them. Also if there are hooks for the grid border? 
![Snag_6fe3a28](https://github.com/microsoft/azuredatastudio/assets/6305833/46578c1c-ec35-4d89-b223-24622b66fd12)
